provider/namespace_type_filter.go

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
filename: namespace_type_filter.go -> aws_namespace_type_filter.go

Reply:
Will be fixed 👍